db/organizations/groove_networks.eno

```diff
@@ -3,7 +3,7 @@ website_url: http://www.groovehq.com/
 privacy_policy_url: https://www.groovehq.com/our/privacy
 privacy_contact: support@groovehq.com
 country: 
-description: "Groove’s dashboard puts your email, live chat, Twitter and Facebook communication in one place, making customer support management hassle-free."
+description: "Groove’s dashboard puts your email, live chat, X and Facebook communication in one place, making customer support management hassle-free."
```

Comment:
Perhaps using `X (Twitter)` in a context like this would make sense? For entries that are not directly related to X (e.g. groove, gumroad, olapic).